docs/index.md

```diff
@@ -0,0 +1,28 @@
+VFS for Git: Virtualized File System for Git
+============================================
+
+VFS stands for Virtual File System. VFS for Git virtualizes the file system
+beneath your Git repo so that Git and all tools see what appears to be a
+normal repo, but VFS for Git only downloads objects as they are needed.
+VFS for Git also manages the files that git will consider, to ensure that
+Git operations such as `status`, `checkout`, etc., can be as quick as
+possible because they will only consider the files that the user has
+accessed, not all files in the repo.
+
+Installing on Windows

```

Comment:
If we're dropping all references to the Mac version, we can simplify this section heading.